src/Symfony/Bundle/DebugBundle/DependencyInjection/Configuration.php

```diff
@@ -26,7 +27,9 @@ public function getConfigTreeBuilder(): TreeBuilder
         $treeBuilder = new TreeBuilder('debug');
 
         $rootNode = $treeBuilder->getRootNode();
-        $rootNode->children()
+        $rootNode
+            ->documentationUri('https://symfony.com/doc/'.(preg_match('/\d+.\d+/', InstalledVersions::getVersionRanges('symfony/debug-bundle'), $m) ? $m[0] : 'current').'/reference/configuration/debug.html')
```

Comment:
`InstalledVersions::getVersion()` would be more correct. `getVersionRanges` return a list of version contraints separated by `||`.

Note that if the package is not installed using Composer, or this package name is not installed (`symfony/symfony` installed, or a fork that changed the name), then this throw a `OutOfBoundsException`.

`documentationUri` could embed a logic for replacing `{version:minor}` and handle exceptions.

Reply:
I like the placeholders solution. I'm always afraid to use `InstalledVersions::getVersion()` because it returns null in a few cases but we should be find here.

Tell me what you think, I implemented your idea in the new method of `NodeDefinition` 🙂